### ISSUE TITLE
confused with red-pitaya-alpine-3.18-armv7-20240204.zip and red-pitaya-debian-12.4-armhf-20240204.zip

### ISSUE THREAD

**Issue by zkf0100007** [pavel-demin/red-pitaya-notes#1148]
I am confused with red-pitaya-alpine-3.18-armv7-20240204.zip and red-pitaya-debian-12.4-armhf-20240204.zip.
What's the different between them?
As described in your example such as https://pavel-demin.github.io/red-pitaya-notes/sdr-receiver/, I built from source, and got red-pitaya-debian-12-armhf.img

How to build a red-pitaya-alpine image?

Thanks!

**Comment by pavel-demin** [comment]
> What's the different between them?

You are not giving me enough context for me to understand what you do not understand, so I do not know how to answer this question.

> How to build a red-pitaya-alpine image?

I use exactly the same commands as in the building from source instructions:
```
source /opt/Xilinx/Vitis/2023.1/settings64.sh
git clone https://github.com/pavel-demin/red-pitaya-notes
cd red-pitaya-notes
source helpers/build-all.sh
```

**Comment by zkf0100007** [comment]
Sorry, I didn't see it.
Both red-pitaya-alpine-3.20-armv7-20240918.zip and red-pitaya-debian-12-armhf.img were generated use the commands as in the building from source instructions.
I know the usage of red-pitaya-alpine-3.20-armv7-20240918.zip
I don't konw why you generate 'red-pitaya-debian-12-armhf.img', and what do you use it for?

Thanks!

**Comment by pavel-demin** [comment]
In the past, I used SD card images based on different Linux distributions. At first I tried Debian Linux and Ubuntu Linux. Then I switched to Alpine Linux.

Currently I mainly use Alpine Linux based SD card images.

I continue to update the script to create a Debian Linux based SD card image as some people are still interested in using it. I do not really use Debian Linux based SD card images myself.
